# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/trusted-types/WorkerGlobalScope-importScripts.html.ini
# Commit: 178d62ae39b5
# Full Hash: 178d62ae39b57dcc9c17ccc8130c6f57d2ecc289
# Author: Mirko Brodesser <mbrodesser@igalia.com>
# Date: 2024-06-25 21:32:30
# Description:
#   Bug 1903879: check inner window's nullness in `TrustedTypePolicyFactor::createPolicy`. r=tschuster
#   
#   The inner window may be null for workers.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D214708
# ==============================================================================

diff -r c6d2db406739 -r 178d62ae39b5 testing/web-platform/meta/trusted-types/WorkerGlobalScope-importScripts.html.ini
--- a/testing/web-platform/meta/trusted-types/WorkerGlobalScope-importScripts.html.ini	Tue Jun 25 08:43:14 2024 +0000
+++ b/testing/web-platform/meta/trusted-types/WorkerGlobalScope-importScripts.html.ini	Tue Jun 25 08:43:14 2024 +0000
@@ -1,2 +1,36 @@
 [WorkerGlobalScope-importScripts.html]
-  expected: CRASH
+  [importScripts with untrusted URLs throws in dedicated worker]
+    expected: FAIL
+
+  [null is not a trusted script URL throws in dedicated worker]
+    expected: FAIL
+
+  [importScripts with two URLs, both strings, in dedicated worker]
+    expected: FAIL
+
+  [importScripts with two URLs, one trusted, in dedicated worker]
+    expected: FAIL
+
+  [importScripts with untrusted URLs and default policy works in dedicated worker]
+    expected: FAIL
+
+  [importScripts with one trusted and one untrusted URLs and default policy works in dedicated worker]
+    expected: FAIL
+
+  [importScripts with untrusted URLs throws in shared worker]
+    expected: FAIL
+
+  [null is not a trusted script URL throws in shared worker]
+    expected: FAIL
+
+  [importScripts with two URLs, both strings, in shared worker]
+    expected: FAIL
+
+  [importScripts with two URLs, one trusted, in shared worker]
+    expected: FAIL
+
+  [importScripts with untrusted URLs and default policy works in shared worker]
+    expected: FAIL
+
+  [importScripts with one trusted and one untrusted URLs and default policy works in shared worker]
+    expected: FAIL